internal/provider/provider.go

```diff
@@ -23,6 +23,10 @@ type provider struct {
 	// provider is built and ran locally, and "test" when running acceptance
 	// testing.
 	version string
+
+	apiEndpoint string
```

Comment:
Small tricky to be able to set up provider configuration for acceptance test. Terraform framework doesn't support default values for provider attributes yet, once they have it implemented we should be able to fetch these values from environment variables.